src/accountsdb/db.zig

```diff
@@ -3958,7 +3954,11 @@ pub const BenchmarkAccountsDBSnapshotLoad = struct {
         const dir_path = sig.TEST_DATA_DIR ++ "bench_snapshot/";
         var snapshot_dir = std.fs.cwd().openDir(dir_path, .{ .iterate = true }) catch {
             std.debug.print("need to setup a snapshot in {s} for this benchmark...\n", .{dir_path});
-            return sig.time.Duration.fromNanos(0);
+            const zero_duration = sig.time.Duration.fromNanos(0);
+            return .{
+                .load_time = zero_duration.asNanos(),
+                .validate_time = zero_duration.asNanos(),
+            };
```

Comment:
this one is very specific and expensive - in the case where you want to run all benchmarks, imo we dont want to error out and stop all of them just because they dont want to run this one